pom.xml

```diff
@@ -331,19 +318,10 @@
           </execution>
         </executions>
         <configuration>
-          <skipITs>${skipIts}</skipITs>
```

Comment:
this will cause all the integration tests to run on CI and locally even if there is no GCP bucket or the like to test against?
possibly you want to rename the property to `skipITs` ?